run.sh

```diff
@@ -2,4 +2,4 @@
 # Dev Server
 #. venv/bin/activate && python manage.py runserver
 # Prod Server
-. venv/bin/activate && gunicorn -b 0.0.0.0:8000 MobSF.wsgi:application --workers=1 --threads=10 --timeout=1800
\ No newline at end of file
+. venv/bin/activate && gunicorn -b 127.0.0.1:8000 MobSF.wsgi:application --workers=1 --threads=10 --timeout=1800
```

Comment:
Keep `-b` default as 0.0.0.0:8000. Allow user to set `ip:port` as first argument for localhost and document it as default in docs.